app/src/main/java/io/branch/branchlinksimulator/RoundTripStore.kt

```diff
@@ -70,6 +70,7 @@ class RoundTripStore(context: Context) : IBranchLoggingCallbacks {
 
     private suspend fun processLog(log: String?) {
         if (log == null) return
+        println(log)
```

Comment:
@gdeluna-branch Do you want to keep this log in Release?

Reply:
Yeah this was already in release a few lines below.